zerver/openapi/zulip.yaml

```diff
@@ -926,7 +942,44 @@ paths:
                                   enum:
                                     - add
                                 person:
-                                  $ref: "#/components/schemas/User"
+                                  allOf:
+                                    - $ref: "#/components/schemas/UserBase"
+                                    - additionalProperties: false
+                                      properties:
+                                        email: {}
+                                        is_bot: {}
+                                        avatar_url:
+                                          nullable: true
+                                        avatar_version: {}
+                                        full_name: {}
+                                        is_admin: {}
+                                        is_owner: {}
+                                        is_billing_admin: {}
+                                        role: {}
+                                        bot_type:
+                                          nullable: true
+                                        user_id: {}
+                                        bot_owner_id:
+                                          nullable: true
+                                        is_active: {}
+                                        is_guest: {}
+                                        timezone: {}
+                                        date_joined: {}
+                                        delivery_email:
+                                          nullable: true
+                                        profile_data: {}
+                                        email_address_visibility:
+                                          type: integer
+                                          description: |
+                                            Which users in this organization can see real email addresses of the user.
+
+                                            - 1 = everyone
+                                            - 2 = members only
+                                            - 3 = administrators only
+                                            - 4 = nobody
+                                            - 5 = moderators only
+
+                                            **Changes**: New in Zulip 5.0 (feature level 119).
```

Comment:
Added this and not using `components/schemas/EmailAddressVisibility` here because we need not need `replacing the realm-level setting` part in the changes entry here.